runtime/doc/lua.txt

```diff
@@ -3074,6 +3095,8 @@ vim.fs.parents({start})                                     *vim.fs.parents()*
         (`string?`)
 
 vim.fs.rm({path}, {opts})                                        *vim.fs.rm()*
+    WARNING: This feature is experimental/unstable.
```

Comment:
Unreleased APIs are marked as experimental. This means `make doc` is needed when doing a release...